.gitignore

```diff
@@ -14,4 +14,5 @@ skaffold.yaml
 .tox
 .pytest_cache
 .overrides
-deployment.yaml
\ No newline at end of file
+deployment.yaml
+/deployment-configuration
```

Comment:
what is the reason to add this to .gitignore?

Reply:
cloudharness does not have a deployment-configuration but it is sometimes useful to specify one locally to do some testing (this was the case)